src/renderer/src/pages/catalogue/catalogue.tsx

```diff
@@ -34,13 +34,34 @@ export function Catalogue() {
     navigate(`/game/${game.shop}/${game.objectID}`);
   };
 
+  const [paginationSettings, setPaginationSettings] = useState({ resultsPerPage: 30 });
+
+  useEffect(() => {
+    async function fetchUserPreferences() {
+      try {
+        const [, userPreferences] = await Promise.all([
+          window.electron.getDefaultDownloadsPath(),
```

Comment:
do you need to call getDefaultDownloadsPath? you dont seem to be using its result